website/source/layouts/layout.erb

```diff
@@ -46,17 +46,6 @@
     <main><%= yield %></main>
     <hashi-footer _data="<%= encode(dato.global_footer) %>"></hashi-footer>
 
-    <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-      ga('create', 'UA-101788060-1', 'auto');
```

Comment:
Same question about whether this value could be used maliciously. It's been _forever_ since I worked with Google Analytics and I don't remember.

Reply:
The same thing applies to GA tracking IDs.

Reply:
Perfect! Thanks!